plugin.xml

```diff
@@ -17,7 +17,8 @@
     <platform name="ios">
       <header-file src="src/ios/Intercom.h" />
       <source-file src="src/ios/libIntercom.a" framework="true" compiler-flags="-ObjC" />
```

Comment:
Hi there. I just did a like for like replacement of what was there before. My ios knowledge is limited so I have probably not done it in the best way here.